spec/inject_spec.rb

```diff
@@ -0,0 +1,66 @@
+
+require './lib/inject'
+
+describe Array do
+  
+  context 'with a block' do
+
+    it 'returns the sum of some numbers starting with the first number in the array' do
+
+      expect(test.inject {|sum, n| sum + n } ).to eq(10) 
+      expect(test.new_inject {|sum, n| sum + n } ).to eq(10) 
+
```

Comment:
Try to avoid having multiple expectations in a test - the result of the first inject can be saved to a variable which you can check against in the expectation on line 11